collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+This release is built from commit [``](https://github.com/godotengine/godot/commit/) (see [README](https://github.com/godotengine/godot-builds/releases/download/4.2-beta1/README.txt)).
+
+- [Breaking changes](#breaking-changes)
+- [Core and systems](#core-and-systems)
+- [2D and 3D features](#2d-and-3d-features)
+- [Editor and usability](#editor-and-usability)
+- [GDExtension](#gdextension)
+- [GUI and theming](#gui-and-theming)
+- [Import and asset pipeline](#import-and-asset-pipeline)
+- [Input](#input)
+- [Multiplayer and networking](#multiplayer-and-networking)
+- [Porting and platforms](#porting-and-platforms)
+- [Rendering, particles, and shaders](#rendering-particles-and-shaders)
+- [Scripting](#scripting)
+- [XR](#xr)
+
+
+### Breaking changes
+
+- The new `AnimationMixer` node is added as an intermediate class for `AnimationPlayer` and `AnimationTree` ([GH-80813](https://github.com/godotengine/godot/pull/80813)).
+- `GraphEdit` and `GraphNode` nodes are reworked with multiple renames, temporary removal of comment nodes, and introduction of `GraphElement` ([GH-79307](https://github.com/godotengine/godot/pull/79307), [GH-79308](https://github.com/godotengine/godot/pull/79308), [GH-79311](https://github.com/godotengine/godot/pull/79311), [GH-81582](https://github.com/godotengine/godot/pull/81582)).
+- An optimization that required splitting raster barriers into vertex and fragment components changes relevant enumerations ([GH-77420](https://github.com/godotengine/godot/pull/77420)). It's unlikely that this affects your project, unless you rely on specific numeric values and store them somewhere.
+- Renaming an audio bus no longer emits the `bus_layout_changed` signal in `AudioServer`; use the new `bus_renamed` signal ([GH-81641](https://github.com/godotengine/godot/pull/81641)).
+- Default base part of the Android package name was changed from `org.godotengine` to `com.example` ([GH-80761](https://github.com/godotengine/godot/pull/80761)).
+
+
+### Core and systems
+
+Following big changes to resource loading and multithreading done in Godot 4.1, the core team is more focused on stability this time around.
+
+This includes addressing design limitations when changing scenes or renaming nodes during `ready`, which led to crashes ([GH-78988](https://github.com/godotengine/godot/pull/78988), [GH-78706](https://github.com/godotengine/godot/pull/78706)), as well as various problems related to renaming and moving files around ([GH-80503](https://github.com/godotengine/godot/pull/80503), [GH-81657](https://github.com/godotengine/godot/pull/81657)). The latter could also lead to corruption of scene files, which should no longer occur. As should not occur sporadic changing of resources IDs in scenes ([GH-65011](https://github.com/godotengine/godot/pull/65011)). There is still room for improvement, but this already makes 4.2 way more version control friendly.
```

Comment:
```suggestion
This includes addressing design limitations when changing scenes or renaming nodes during `ready`, which led to crashes ([GH-78988](https://github.com/godotengine/godot/pull/78988), [GH-78706](https://github.com/godotengine/godot/pull/78706)), as well as various problems related to renaming and moving files around ([GH-80503](https://github.com/godotengine/godot/pull/80503), [GH-81657](https://github.com/godotengine/godot/pull/81657)). The latter could also lead to corruption of scene files, which should no longer occur. Sporadic changing of resources IDs in scenes should also no longer occur ([GH-65011](https://github.com/godotengine/godot/pull/65011)). There is still room for improvement, but this already makes 4.2 way more version control friendly.
```
Made it an actual sentence.

Reply:
Going with Clay's suggestion.

Reply:
Yep, he beat me to it and his wording is better :)